### ISSUE TITLE
Are proxies no longer working?

### ISSUE THREAD

**Issue by heartofalion** [Stichoza/google-translate-php#107]
Since proxies are essential these days to make a significant amount of translation requests, I've tried to use a remote proxy following the Readme. However I've tried just about everything at this point, but whatever I try to do in order to send translation requests via a proxy, it fails. I'm hoping someone might be able to see what I'm doing wrong:

```
$tr = new GoogleTranslate("en", "de", [ 
            'timeout' => 10,             
            'proxy' => [  
                'http'  => 'tcp://xxx.xxx.xxx.xxx:80'               
            ],
            'headers' => [
                'User-Agent' => 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36'
            ]
        ]);
```

Can someone post an example of code that makes a succesful connection to a remote proxy?

**Comment by frzsombor** [comment]
If you are using only one proxy, it is no different than using your server's real IP. If you trigger an alarm with that single IP, it will be banned, and you are done. If you have to send that many requests, you will need a lot of proxies. And you will need high quality ones, not some already black listed ones. I can't help you finding these, but Google will. Buy 10-20-50-100 (depending on your needs - and budget), and for every request, select one randomly, and use that to send the request. This way you will balance the traffic evenly between those IPs, and if you found out the correct amount, hopefully none of them will get banned.

**Comment by heartofalion** [comment]
Yes, load-balancing the requests over multiple proxies was my intention. It's not ideal, but the lack of transliteration on the paid version of Google Cloud translate gave me no other option.

I thought I'd give an update in regards to how I resolved the transliteration issue, as many other people might run into this issue as well.

If you want to do transliteration, but you ran out of requests via Stichoza, then you have the following options:

1. PHP has an extension called 'PHP Intl'. This extension deals with internationalization of PHP related code. This extension, when installed and enabled in php.ini, has a function called: transliterator_transliterate(). This function allows you to do transliteration of almost every human language script. The great thing is that it's free and pretty much native to the PHP language.

2. PHP transliteration libraries for some languages are available on packagist.org .  These packages are free and open source.

3. Microsoft Azure Translator. Just like Google and Amazon, Microsoft Azure has a translation API. The Azure Translator API has support for transliteration. The downside is that it costs money. However I have found that Microsoft Azure Translator is the least expensive of the three. Microsoft currently charges $10 USD per million characters, while Amazon charges $15 USD per million characters and Google $20 USD per million characters. 

4. Write your own transliteration functions. For example the Russian alphabet has 33 characters and you can easily write a function for it using an array containing the romanized Latin characters and Cyrillic characters as a counterpart. You can do this for both upper and lowercase letters - and voila, you have yourself a transliteration function. This however is not so easy for languages like Chinese due to the enormous amount of characters, but for languages like Russian, Greek, etc, writing your own transliteration function is a solid option.

**Comment by neobie** [comment]
I think the proxy options is not working
